lib/puppet/provider/cisco_interface/nxapi.rb

```diff
@@ -55,9 +55,15 @@
     :switchport_trunk_native_vlan,
   ]
   INTF_BOOL_PROPS = [
-    :shutdown, :negotiate_auto, :ipv4_redirects, :ipv4_proxy_arp,
-    :switchport_vtp, :switchport_autostate_exclude,
-    :svi_autostate, :svi_management
+    :shutdown,
```

Comment:
If this ordering is necessary here, can you add a comment to that effect? If not, can you put these into alphabetical order?